### ISSUE TITLE
scp cant find file

### ISSUE THREAD

**Issue by Mosch0512** [edvin/fxlauncher#155]
Mybe i am just to stupid ...
Am i doing sth wrong?

so i am doing sth like that.

```groovy
fxlauncher {
    applicationVendor 'Someone'
    // Base URL where you will host the application artifacts
    applicationUrl "https://www.google.com/"
    applicationMainClass 'com.fun.MyApp'
    acceptDowngrade false
    // Optional scp target for application artifacts hosted at the above url
    deployTarget "C:/test"
}
```

Questions I have:
1. Why is applicationUrl required
2. What if i dont have an applicationUrl

Path variable is set to:
```
C:\something\OpenSSH
```

Error i get while running deployApp:
```gradle
* What went wrong:
Execution failed for task ':deployApp'.
> java.io.IOException: Cannot run program "scp": CreateProcess error=2, The system cannot find the file specified
```

But i can use it from the command line...
```bash
D:\Users\someuser>scp
usage: scp [-346BCpqrv] [-c cipher] [-F ssh_config] [-i identity_file]
           [-l limit] [-o ssh_option] [-P port] [-S program] source ... target
```

**Comment by ronsmits** [comment]
the applicationURL is where the actual app is stored. This is where
fxlauncher will look for updates. SCP not found might be a $PATH issue

**Comment by Mosch0512** [comment]
Okay thx so far :)

With updates you mean sth like if new version of MyApp is available it will replace current with new one?
Can i also give an file uri? for the applicationURL?

And the SCP Path think should be correct i hope so ... 

![grafik](https://user-images.githubusercontent.com/48485443/55067037-45956c80-507f-11e9-93d4-9e1923afe5bc.png)

**Comment by Mosch0512** [comment]
It is working now fine... after restarting my machine xD

**Comment by UgmaDevelopment** [comment]
For anyone passing by:
I was having this same issue on Windows 10. I could run `scp` on the command line, but when I ran the `deployApp` task in the [Gradle ](https://www.google.com/search?client=firefox-b-1-d&q=gradle)[FXLauncher Plugin](https://github.com/edvin/fxlauncher-gradle-plugin), I kept getting `Cannot run program "scp": CreateProcess error=2, The system cannot find the file specified`. I had also restarted various times to no avail.

I looked at my path and realized that I had multiple references to various `scp.exe`s on my path.

I tested and found that—at least for me—Windows 10's copy in `C:\Windows\System32\OpenSSH` would not work for me, even if it did on the command line when just running `scp`.

I found that removing `C:\Windows\System32\OpenSSH` (and/or `%SYSTEMROOT%\System32\OpenSSH\`) from my path and instead using the scp that comes when you install git, solved the issue for me. Once git is installed that scp is located at `C:\Program Files\Git\usr\bin`. Once I added just that one to my path it worked.
